src/MiniCover.HitServices/TestMethodUtils.cs

```diff
@@ -31,10 +44,15 @@ private static bool HasPdb(MethodBase methodBase)
             lock (Lock)
             {
                 if (AssemblyHasPdbCache.TryGetValue(location, out var hasPdb)) return hasPdb;
-                hasPdb = File.Exists(Path.ChangeExtension(location, ".pdb"));
+                hasPdb = IsNotATestFrameworkAssembly(methodBase.DeclaringType.Assembly) && File.Exists(Path.ChangeExtension(location, ".pdb"));
                 AssemblyHasPdbCache.Add(location, hasPdb);
                 return hasPdb;
             }
         }
+
+        private static bool IsNotATestFrameworkAssembly(Assembly assembly)
+        {
+            return !TestFrameworkAssemblies.Any(framework => assembly.GetName().Name.Equals(framework, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Since this method might be called several times in some situations, maybe it's better to use a Hashset O(1) and lose the IgnoreCase comparison.

Reply:
Agree